.sbtopts

```diff
@@ -1,3 +1,2 @@
 -J-Xmx3072M
 -J-Xms1024M
--J-XX:MaxMetaspaceSize=512M
```

Comment:
did something change in sbt, or just general cleanup?
(we have been struggling with full meta space GC pauses on Jenkins, but I'm not sure that was the reason for setting this)

Reply:
I think there was a build that ran out... instead of removing it we could also increase it, of course, but that seemed a bit arbitrary